### PR TITLE
Adding scripts for rollout test

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Add scripts for running rollout test from upstream in rh-che repo via periodical jenkins job. 

### What issues does this PR fix or reference?
https://github.com/redhat-developer/che-functional-tests/issues/306

### How have you tested this PR?
Locally against development cluster.

### CODE REVIEW

**Comment by Katka92** [comment]
@ScrewTSW please could you have a look if there is something which should prevent this job to run correctly? Like dependencies etc. Thanks!

File: .ci/cico_rollout_test.sh [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+#This script expects this environment variables set:
+# CHE_TESTUSER_NAME, CHE_TESTUSER_PASSWORD, CHE_TESTUSER_EMAIL, RH_CHE_AUTOMATION_DEV_CLUSTER_SA_TOKEN
+
+
+# --- SETTING ENVIRONMENT VARIABLES ---
+export PROJECT=testing-rollout
+export CHE_INFRASTRUCTURE=openshift
+export CHE_MULTIUSER=true
+export CHE_OFFLINE_TO_ACCESS_TOKEN_EXCHANGE_ENDPOINT=https://auth.prod-preview.openshift.io/api/token/refresh
```
Comment (ScrewTSW) [inline .ci/cico_rollout_test.sh]: Didn't this part change? That both API endpoints are set from a common variable?

File: .ci/cico_rollout_test.sh [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                                         -z \
+                                         -U;
+then
+  echo "Che successfully deployed."
+else
+  echo "Custom che deployment failed. Error code:$?"
+  exit 4
+fi
+
+curl https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz -o oc.tar.gz
```
Comment (ScrewTSW) [inline .ci/cico_rollout_test.sh]: I would replace the static version number in the URL with a variable. And isn't 3.9. binary already outdated?

Comment (ScrewTSW) [reply .ci/cico_rollout_test.sh]: e.g.:
```
export OC_VERSION=3.9.33
curl -s "https://mirror.openshift.com/pub/openshift-v3/clients/${OC_VERSION}/linux/oc.tar.gz" | tar xvz -C /usr/local/bin
```

**Pull request merged**